packages/veritone-widgets-server/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "veritone-widgets-server",
+  "version": "1.0.0",
+  "description": "Backend server for widgets",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Andrew Tung",
+  "license": "ISC",
```

Comment:
update to Apache-2.0 (and would you also please add the `LICENSE` file and disclaimer to the bottom of the `README.md`? copy them from another package)